aws/resource_aws_network_acl.go

```diff
@@ -303,13 +303,16 @@ func resourceAwsNetworkAclUpdate(d *schema.ResourceData, meta interface{}) error
 					}
 					return fmt.Errorf("Failed to find acl association: acl %s with subnet %s: %s", d.Id(), r, err)
 				}
-				log.Printf("DEBUG] Replacing Network Acl Association (%s) with Default Network ACL ID (%s)", *association.NetworkAclAssociationId, *defaultAcl.NetworkAclId)
+				log.Printf("[DEBUG] Replacing Network Acl Association (%s) with Default Network ACL ID (%s)", *association.NetworkAclAssociationId, *defaultAcl.NetworkAclId)
 				_, err = conn.ReplaceNetworkAclAssociation(&ec2.ReplaceNetworkAclAssociationInput{
 					AssociationId: association.NetworkAclAssociationId,
 					NetworkAclId:  defaultAcl.NetworkAclId,
 				})
 				if err != nil {
-					return err
+					if isAWSErr(err, "InvalidAssociationID.NotFound", "") {
+						return nil
```

Comment:
This should be `continue` so the `for` loop and the rest of the `resourceAwsNetworkAclUpdate` function will process. 👍 

```suggestion
						continue
```

Reply:
(or update the `if err != nil` line to `if err != nil && !isAWSErr(err, "InvalidAssociationID.NotFound", "")`)

Reply:
Good catch thanks for your feedback 👍